### PR TITLE
Add option to convert quantity to astropy quantity

### PR DESCRIPTION
**Pull request by rieder**
This PR adds an "as_astropy_quantity" method to AMUSE quantities if astropy is available and can be imported.
In case it is not available, it simply doesn't add the method.

### CODE REVIEW

**Comment by rieder** [comment]
This would (partially) solve #170.

**Comment by rieder** [comment]
ok to merge?

File: src/amuse/units/quantities.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
 from amuse.units.si import none
 from amuse.units.core import zero_unit
 
+try:
+    import astropy.units
+    from amuse.units import units
+    astropy_available = True
```
Comment (ipelupessy) [inline src/amuse/units/quantities.py:18]: usually this kind of variable (global in the file is denoted by some combination of capitals and underscores...), so:
__HAS_ASTROPY__=True

**Comment by ipelupessy** [comment]
apart from the small stylistic comment, ok

File: src/amuse/units/quantities.py (hunk 1 above)
Comment (rieder) [reply src/amuse/units/quantities.py]: done

**Comment by ipelupessy** [comment]
one thing I figured just now is that if you don't have astropy installed, you get an atrribute error, maybe its better to raise "install astropy/ astropy not found error" inside the to_astropy function instead of defining anly when astropy is present...

**Comment by rieder** [comment]
sure

**Pull request merged**